src/app/datagrid/hide-show-columns/hide-show.html

```diff
@@ -12,49 +12,49 @@
 <button class="btn btn-outline-primary" (click)="toggleDate()">
     Toggle Date
 </button>
-<clr-datagrid>
-    <clr-dg-column *ngIf="showId">
-        <ng-container *clrDgHideableColumn>
-            User ID
-        </ng-container>
-    </clr-dg-column>
-    <clr-dg-column>
-        <!--Name-->
-        <ng-container *clrDgHideableColumn>
-            Name
-        </ng-container>
-    </clr-dg-column>
-    <clr-dg-column *ngIf="showDate">
-        <ng-container *clrDgHideableColumn>
-            Creation date
-        </ng-container>
-    </clr-dg-column>
-    <clr-dg-column>
-        <ng-container *clrDgHideableColumn>
-            Pokemon
-        </ng-container>
-    </clr-dg-column>
-    <clr-dg-column>
-        <ng-container *clrDgHideableColumn>
-            Favorite color
-        </ng-container>
-    </clr-dg-column>
+    <clr-datagrid>
```

Comment:
🤔  - I didn't catch that when I created the PR. Let me see if I can get rid of them.